app/src/components/molecules/ActivityMarker/index.jsx

```diff
@@ -27,6 +28,8 @@ export const ActivityMarker = ({
     const selectedType = types[activityType];
     const icon = getActivityIcon(activityType);
     const isRiskGroup = activity.user?.riskGroup?.length > 0;
+    const markerColor = isRiskGroup ? 'bg-danger-darker' : 'bg-primary-darker';
+    const isFocused = focused ? markerColor : 'bg-white';
```

Comment:
Opa, alteração só pra deixar as variáveis mais descritivas
```suggestion
    const markerDefaultColor = isRiskGroup ? 'bg-danger-darker' : 'bg-primary-darker';
    const currentMarkerColor = focused ? markerDefaultColor : 'bg-white';
```

Reply:
feito